docs/source/software/quick_start_guide.rst

```diff
@@ -105,4 +105,51 @@ QGroundControl and change the following:
 * EKF2_BARO_GATE = 0
 * EKF2_EVP_NOISE = 0.01
 * EKF2_EV_GATE = 500
-* EKF2_HGT_MODE = vision
\ No newline at end of file
+* EKF2_HGT_MODE = vision
+
+
+Safety Checklist
+==================
+
+.. important::
+
+    Make sure you complete the following check list *before every flight*. Most
+    of the checks are automated by the controller on startup, but that is no
+    substitute for good ole human caution.
+
+
+Hardware
+----------
+
+1. Mechanical
+
+    a. All *4* rotors are tightly secured (more than hand tight)
+    b. Battery is secured
+
+2. Electrical
+
+    a. The fcu serial connection is secured between the px4 and cc
+    b. Battery voltage is over the nominal 11.5v (*DO NOT* start a flight with
+       less than this voltage)
+
+
+Software
+--------
+
+1. Motive
+
+    a. z is up
+    b. quad frame is aligned with room (room's long-axis is quad x-axis)
+
+2. ROS Topics
+
+    a. ``local_position/pose`` matches ``vision_pose/pose``
+    b.  ``state`` has ``connected=True``
+
+3. PX4 Arms
+
+    a. Arm the quad but *DO NOT* takeoff as you will not have attitude control
+    b. This checks that the PX4 is correctly communicating with the onboard
+       sensors
+    c. This can be checked without even ssh-ing the quad
+    d. Do not try to takeoff, just arm check propeller rotation and disarm
```

Comment:
> I think the above Software 1. and 2. look good, but I'm not sure I specifically follow what you mean by 3. I think some elaboration on each of these points might be useful.
> 
> I think this checklist is great for running through to verify safety before flying. However, for all the points in this checklist, would it be beneficial to have an elaborated page of the safety guide? There, maybe pictures or elaboration could be useful for someone who has never used the system before. This could include the comment I made earlier about the meaning of "secured".

Referring back to my last comment on this section. I think this is more clear than the previous review, but I think I'm still a bit confused on all the details of 3. Maybe it's just because I haven't done this before, but I want to make sure others agree this looks good to go or if we need a bit of elaboration. Either in the form of a detailed checklist (maybe something that could even be printed to check off) or just some elaboration in the a-d points. What do you think @lowjunen @cacadam89 ?

Reply:
basically i just want the person to arm the quad on the ground, this will spin up the props but not enough for takeoff. this arming check is a good way to just see if the px4 system is in a good state (i.e. no sensor issues).

Reply:
I think a check on the kill switch should be added. Something like:

PX4 Arms and Kills
a. Arm the quad by pinching the sticks down and inwards. *DO NOT* push up throttle stick after arm as this will cause the craft to take off.
b. gently push the throttle stick up until the motors start spinning. check that the propeller rotation is in the correct direction.
c. disarm the quad by pinching the sticks down and inwards.
d. Arm the quad again but this time, use the kill switch to disarm. The motors should cut instantaneously.

Note: For extra safety, you are advised to run these checks without the propellers attached.